packages/components/src/components/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './layouts';
 export * from './primitives';
+export * from './notifications';
```

Comment:
Should we keep `Toast` inside of primitives and maybe restructure the components in a different PR? @tkanzakic `Toast` is considered a `Feedback` component in chakra ui

Reply:
Renaming as `feedback` makes more sense to me as well, but the Figma files have it as `notifications`. @tkanzakic thoughts?